app/controllers/concerns/is_logged_in_with_any_method.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Concerns
+  module IsLoggedInWithAnyMethod
+    extend ActiveSupport::Concern
+
+    # rubocop:disable Rails/LexicallyScopedActionFilter
+    included do
+      before_action :verify, except: %i[interactive interactive_render]
+    end
+    # rubocop:enable Rails/LexicallyScopedActionFilter
+
+    private
+
+    def verify
+      result = current_user
```

Comment:
current_user is gedefinieerd in application helper en daar kijk je ook al naar of je via jwt_token kan authen, en dat gaat dan volgens mij goed, maar vervolgens returnt ie de auth_user, en als je nog niet authenticate_auth_user hebt aangeroepen, dan bestaat die nog niet dus dan roep je authenticate_auth_user aan en dan maakt ie een auth_user van de payload en een person en vanaf dat moment returnt current_user wel de person van de auth_user?

Man ik breek hier mijn hoofd over. Maar volgens mij klopt het allemaal wel.

Maar het heeft wel een side effect dat misschien niet zo netjes is. Die JWT authenticator#auth methode slaat namelijk iets op in de cookies voordattie kijkt of de auth user wel bestaat. Dat is misschien niet zo netjes (misschien zou je die volgorde om kunnen draaien).

Reply:
ik heb ze omgedraaid.